ios-trivia-game/ViewControllers/ResultsViewController.swift

```diff
@@ -32,4 +33,15 @@ class ResultsViewController: UIViewController {
     }
     */
 
+    @IBAction func onQuitFromResults(_ sender: UIBarButtonItem) {
+        // update user_in_game to remove player from user_in_game
+        
+        FirebaseClient.instance.quitGame(complete: {
```

Comment:
The function looks exactly the same to me, which indicates that we're using it the exact same way in several places. The behavior seems to be, quit the game, and then return to the main tab controller.